### ISSUE TITLE
Incorporating this into SPMT or equivalent experience

### ISSUE THREAD

**Issue by pkbullock** [pnp/sp-dev-modernization#491]
#### Category

[ ] Bug
[X] Enhancement

#### Expected or Desired Behavior

Include the modernization tooling as part of SPMT or Standalone tooling. This would provide a UI like experience to converting the pages - for those that are less familiar with PowerShell or ASP.NET. 

I might do some investigating over the demand for this.

**Comment by jakobnotseth** [comment]
I think this is a great idea and would provide a much better upgrade experience from on-prem calssic to cloud modern.

**Comment by jansenbe** [comment]
@jakobnotseth : Assume we did this, would only supporting wiki, blog and webpart pages still be valuable for you? Publishing page transformation requires to create a pagelayout mapping file, hence slightly more complex.

**Comment by jakobnotseth** [comment]
Not really, as most migration projects is from PublishingPages. But if you could add some default logic to do this with PublishingPages it would be great. Even if the default was to skip webparts that doesn't fit without the mapping-customizations.
